### PR TITLE
lowercase parameters

### PR DESCRIPTION
**Pull request by Razybehbahani**

### CODE REVIEW

**Comment by Razybehbahani** [comment]
Checking TX_type and motivation

File: .ipynb_checkpoints/test-checkpoint.ipynb
Comment (ttricco) [inline .ipynb_checkpoints/test-checkpoint.ipynb:1]: Remove.

File: bankcraft/__pycache__/Motivation.cpython-311.pyc
Comment (ttricco) [inline bankcraft/__pycache__/Motivation.cpython-311.pyc:1]: Remove all the __pycache__ files.

**Review by ttricco — COMMENTED** [review]
Remove unnecessary files.

**Pull request closed without merging**